### ISSUE TITLE
Time functions issue

### ISSUE THREAD

**Issue by simonhoenegger** [guh/guh#410]
Time based rules and the Plugin Time/Alarm don't trigger at configured time.
Raspberry 2
Ubuntu 16.04.1 LTS
VERSION:  0.7.8+20161227

**Comment by simonhoenegger** [comment]
I now changed to the guhio snappy build. Still guhd version 0.7.8 .
Runs great btw!
But regarding the related issue I found out that even though I configured

"timeZone": "Europe/Vienna" (UTC +1)
in guh, the rule engine still triggers in respect to UTC.

I configured a rule with a
TimeDescriptor:
    0. -> Every day at 04:50

which is triggered
05:50 CET

the logs in guh-cli also show the correct CET time:
(I) 2017-02-02 05:50:00 |     Action executed 

Hope this helps finding the issue?
